src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCause.java

```diff
@@ -199,55 +201,6 @@ public FormValidation validate(String newName,
         return FormValidation.ok();
     }
 
-    /**
-     * Form validation for {@link #description}. Checks for not empty and not "Description..."
-     *
-     * @param value the form value.
-     * @return {@link hudson.util.FormValidation#ok()} if everything is well.
-     */
-    public FormValidation doCheckDescription(@QueryParameter final String value) {
```

Comment:
Should keep an `@Deprecated` method signature here for binary compatibility that just calls the new method.
There must have been a reason we put it here in the beginning.